libafl/src/feedbacks/mod.rs

```diff
@@ -438,6 +467,22 @@ where
         let b = second.is_interesting(state, manager, input, observers, exit_kind)?;
         Ok(a || b)
     }
+    #[cfg(feature = "track_hit_feedbacks")]
+    fn last_result(first: &A, second: &B) -> Option<bool> {
+        Some(
+            first.last_result().expect("should have run")
```

Comment:
It might be better to return `None` here and catch this all the way up?

Reply:
Or even make this return a `Result` instead, in general? 99% of the time the bool will be set

Reply:
We shouldn't expect in a library, and it's always `Some` after the first run, so I think the current situation isn't great

Reply:
Fair enough, I'll change it